content/200-orm/100-prisma-schema/20-data-model/50-database-mapping.mdx

```diff
@@ -138,13 +138,7 @@ model Post {
 
 Since no explicit names are provided via `map` arguments Prisma will assume they follow our default naming convention.
 
-### Prisma's default naming conventions for indexes and constraints
-
-If you introspect a database the names for indexes and constraints will be added to your schema unless they follow Prisma's naming convention. If they do, the names are not rendered to keep the schema more readable. When you migrate such a schema Prisma will infer the default names and persist them in the database.
-
-We chose our naming convention to align with PostgreSQL since it is deterministic and helps us maximize the amount of times where we do not need
-to render names because they already align with the convention.
```

Comment:
Was this section intentionally removed? It seems the first paragraph was a repeat, but is the whole headline and section not needed?

Reply:
It reads redundant to me, especially because it's already written in lines 97, 99, and 116. 
Your call though

Reply:
This section is all kinds of broken and weird in its current state already, including incomplete sentences (line 120).

I'll try to untangle that.

Reply:
Good catch, this was an absolute mess indeed but I think I figured it out (the headline with the same text above was faulty as `####`, but as `###` the whole structure makes more sense now. I have added a bunch of suggestions that should fix the whole section.